kafka-versions.yaml

```diff
@@ -26,15 +26,15 @@
   #third-party-libs: 2.3.x
   #default: false
   #unsupported-features: kafkaMirrorMaker2
-- version: 2.3.1
-  format: 2.3
-  protocol: 2.3
-  url: https://archive.apache.org/dist/kafka/2.3.1/kafka_2.12-2.3.1.tgz
-  checksum: B9FD57DE7D9691C89CF668901E5E29728F10249667611A0603C322C8823EC2D5568B51EB1A570D4D9119CD4E44E84B590CAB99B33A0355EBAC37811A7D453608
-  zookeeper: 3.4.14
-  third-party-libs: 2.3.x
-  default: false
-  unsupported-features: kafkaMirrorMaker2
+#- version: 2.3.1
```

Comment:
I think you can just delete this completely, not just comment it out.

Reply:
Of course, will do.